.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,3 +1,12 @@
+---
+name: Bug Report
+about: Create an issue that currently exist with JustAskDavidB
```

Comment:
Change 'exist with' to 'exists in'

Reply:
@andrew-zhouu  this has been resolved. thanks.